### DISCUSSION TITLE
Graph generation not working

### DISCUSSION THREAD

**Discussion by xplosionmind** [manunamz/jekyll-wikilinks#20] [Q&A]
I am opening a discussion thread instead of an issue because probably the problem I am experiencing is not due to a bug but because of my total ignorance of JavaScript.

So, I perfectly replicated the graph code in [this gist](https://gist.github.com/manunamz/3222e73c6b7eaef3a677a26e8f177466 'd3 graph-net-web example '): in [my website](https://tommi.space)’s [source code](https://github.com/xplosionmind/tommi.space 'tommi.space’s source code on GitHub'), [here](https://github.com/xplosionmind/tommi.space/blob/main/jam-graph.js) there is the JavaScript main file, [here](https://github.com/xplosionmind/tommi.space/blob/main/_sass/_jam-graph.scss) there is the original SCSS, and [here](https://github.com/xplosionmind/tommi.space/blob/main/_includes/jam-graph.html) the HTML include. Nevertheless, no graph gets displayed, as you can see in the empty space left for example [here](https://tommi.space/jam#svg-graph). What am I doing wrong?

**Comment by xplosionmind** [comment]
So… does anybody have any idea about this?

**Reply by manunamz** [reply]
Hey -- sorry I just saw this (I was getting issue notifications, but not discussion notifications). I will take a look at this in a bit. It's possible the updates to `jekyll-wikilinks` may have caused the implementation to drift away from the graph code in the gist.

**Reply by manunamz** [reply]
Try changing [this line](https://github.com/xplosionmind/tommi.space/blob/main/jam-graph.js#L3) to match whatever `path` you set for the graph file in your configs.

**Reply by xplosionmind** [reply]
As you can see in [my `_config.yml`](https://github.com/xplosionmind/tommi.space/blob/main/_config.yml), I have no path specified for `d3_graph_data`. hence, the path should match the one in the original file.

**Reply by manunamz** [reply]
Hmm...Ok. I am actually in the middle of making a lot of changes to the graph, so when I release that I will revisit this.
